android/runtime/common/src/js/titanium.js

```diff
@@ -61,13 +61,8 @@ TitaniumWrapper.prototype = Titanium;
 Titanium.Wrapper = TitaniumWrapper;
 
 function UIWrapper(context, Android) {
```

Comment:
<!--
  0 failure: 
  1 warning:  android/runtime/c...
  
  
  DangerID: danger-id-default;
  File: android/runtime/common/src/js/titanium.js;
  Line: 63;
-->

- :warning: android/runtime/common/src/js/titanium.js line 63 – 'Android' is defined but never used. (no-unused-vars)